Sources/Core/ARM/SoftwareBreakpointManager.cpp

```diff
@@ -131,6 +117,8 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
   }
 #endif
 
+  opcode.clear();
```

Comment:
Yeah, not using it just assumes the caller passes a clean argument. Which is fine, but I also don't think you incur any cost from clearing an empty vector. So better to prevent future bugs?